.drone.star

```diff
@@ -3156,3 +3164,63 @@ def pipelineSanityChecks(ctx, pipelines):
 
     for image in images.keys():
         print(" %sx\t%s" % (images[image], image))
+
+def uploadErrorTrace():
```

Comment:
if tracing is enabled by a flag like `[with-tracing]` it's not always an error. Could we rename it to something like `uploadTracingResult`? Not an show stopper just my 5 cents